nosqlbench/http-jsonapi-keyvalue.yaml

```diff
@@ -11,8 +11,8 @@ description: |
 scenarios:
```

Comment:
this file is missing a MD file

Reply:
There isn't one on the equivalent docs API test either. We'll come back and address separately.